security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -527,6 +532,24 @@ func (sc *SecretCache) generateSecret(ctx context.Context, token, resourceName s
 		certChain = append(certChain, []byte(c)...)
 	}
 
+	// Cert exipre time by default is createTime + sc.configOptions.SecretTTL.
+	// Current CAs(Citadel, GoogleCA) respect SecretTTL that passed to it and use it decide TTL of cert it issued.
+	// Some customer CA may override TTL param that's passed to it.
+	expireTime := t.Add(sc.configOptions.SecretTTL)
+	if !sc.configOptions.SkipValidateCert {
+		block, _ := pem.Decode(certChain)
```

Comment:
Replace "_" with "err" and print it out in the log?

Reply:
the second ouput isn't err.